afkak/test/testutil.py

```diff
@@ -162,6 +167,7 @@ def setUp(self):
                 clientId=self.topic)
 
         yield ensure_topic_creation(self.client, self.topic,
+                                    fully_replicated=True,
```

Comment:
Not certain, but I seem to remember some tests where this might be a problem.  That might be unit-test only though.